src/task3.py

```diff
@@ -17,31 +18,31 @@ def __call__(self, *args, **kwargs):
         start_time = time()
         with contextlib.redirect_stdout(f):
             self.function(*args, **kwargs)
-            
 
         end_time = time()
         result = f.getvalue()
         self.dic_time[self.function.__name__] = end_time-start_time
         with open('Task3_Output.txt', 'a+') as f:
-            f.write('{} call {} excuted in {} sec\n'.format(self.function.__name__ , self.count, end_time-start_time))
-            #f.write(f'{self.function.__name__} call {self.count} excuted in {(result):.4f} sec')
+            f.write('{} call {} excuted in {} sec\n'.format(
+                self.function.__name__, self.count, end_time-start_time))
             f.write(f'Name:\t{self.function.__name__}\n')
             f.write(f'Type:\t{type(self.function)}\n')
             f.write(f'Signs:\t{inspect.signature(self.function)}\n')
             f.write(f'Args: \tpositional {args} \n\t\tkey=worded {kwargs}\n\n')
-            f.write(f'Doc:\t{inspect.getdoc(self.function)}'.replace('\n', '\n\t\t')+'\n\n')
-            f.write(f'Source:\t{inspect.getsource(self.function)}'.replace('\n', '\n\t\t')+'\n')
+            f.write(f'Doc:\t{inspect.getdoc(self.function)}'.replace(
+                '\n', '\n\t\t')+'\n\n')
+            f.write(f'Source:\t{inspect.getsource(self.function)}'.replace(
+                '\n', '\n\t\t')+'\n')
             f.write(f'Output:\t{result}'.replace('\n', '\n\t\t') + '\n\n')
-        
+
         return result
 
-    @staticmethod
-    def ranking():
+    @classmethod
```

Comment:
Updated `ranking` to be a `@classmethod`, which makes accessible only to the class object, and can be called without arguments from the class object.